gravitee-am-ui/src/app/domain/settings/password-policies/password-policies-idp-select-dialog/password-policies-idp-select-dialog.component.ts

```diff
@@ -69,17 +63,10 @@ export class PasswordPoliciesIdpSelectDialogComponent {
   }
 
   confirmSelection(): void {
-    this.dialogRef.close(this.result);
+    this.dialogRef.close(this.unlinkedResult);
   }
```

Comment:
confirmSelection closes and accepts 'selection' from linked or unliked tab. Here, I guess linkedResult is missing.

Reply:
it will be handled here: [AM-2895](https://gravitee.atlassian.net/browse/AM-2895)